.github/workflows/bump-tag-publish.yaml

```diff
@@ -65,7 +65,7 @@ jobs:
           echo semver-part=$SEMVER_PART >> $GITHUB_OUTPUT
           echo checkout-branch=$CHECKOUT_BRANCH >> $GITHUB_OUTPUT
       - name: Checkout current code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
v4 actions are needed to address GH action deprecation warnings for nodejs

Reply:
Thanks. I had noticed these warnings but hadn't gotten around to fixing them yet.